src/Altinn.Platform/Altinn.Platform.Storage/IntegrationTest/TestData/MessageBoxInstanceData.cs

```diff
@@ -0,0 +1,311 @@
+using System;
+using System.Collections.Generic;
+using Altinn.Platform.Storage.Models;
+using Storage.Interface.Models;
+
+namespace Altinn.Platform.Storage.IntegrationTest
+{
+    /// <summary>
+    /// Class exposing test data for messageboxinstancecontroller integration tests
+    /// </summary>
+    public class MessageBoxInstanceData
+    {
+        private static string instanceOwnerId_1 = "50002108";
+        private static string userId_1 = "20000000";
+
+        private static string org_1 = "TDD";
+        private static string org_2 = "SPF";
+
+        private static readonly string AppName_1 = "test-applikasjon-1";
+        private static readonly string AppName_2 = "test-applikasjon-2";
+        private static readonly string AppName_3 = "test-applikasjon-3";
+
+        private static readonly string AppId_1 = $"{org_1.ToLower()}/{AppName_1}";
+        private static readonly string AppId_2 = $"{org_1.ToLower()}/{AppName_2}";
+        private static readonly string AppId_3 = $"{org_2.ToLower()}/{AppName_3}";
+
+        private static Dictionary<string, string> appTitles_App1 = new Dictionary<string, string>()
+        {
+            { "nb", "Test applikasjon 1 bokmål" },
+            { "en", "Test application 1 english" },
+            { "nn-NO", "Test applikasjon 1 nynorsk" }
+        };
+
+        private static Dictionary<string, string> appTitles_App2 = new Dictionary<string, string>()
+        {
+             { "nb", "Test applikasjon 2 bokmål" },
+             { "en", "Test application 2 english" }
+        };
+
+        private static Dictionary<string, string> appTitles_App3 = new Dictionary<string, string>()
+        {
+            { "nb", "Test applikasjon 3 bokmål" },
+            { "nn-NO", "Test applikasjon 3 nynorsk" }
+        };
+
+        private static Application app_1 = new Application()
+        {
+            Id = AppId_1,
```

Comment:
better to have application id, org, user id values similar to our real time data, so that it's easily understood

Reply:
I'll updater the testdata ;)